### PR TITLE
improve_hacker_new

### PR DESCRIPTION
**Pull request by LuckyPigeon**
refer to #86 

## Description

Rendering html after retrieved from hacker news

- [x] Local setup
- [x] codesandbox setup

## Original

image

## After

image

### CODE REVIEW

**Comment by dai-shi** [comment]
Preview: https://githubbox.com/LuckyPigeon/jotai/tree/improve_hacker_new/examples/hacker_news

**Review by dai-shi — APPROVED** [review]

File: .size-snapshot.json [hunk 1]
```diff
@@ -1,39 +1,39 @@
 {
   "index.js": {
-    "bundled": 16393,
-    "minified": 7275,
-    "gzipped": 2794,
+    "bundled": 184,
+    "minified": 160,
+    "gzipped": 94,
```
Comment (dai-shi) [inline .size-snapshot.json:5]: Can you revert the change in this file?

**Review by dai-shi — CHANGES_REQUESTED** [review]

File: .size-snapshot.json (hunk 1 above)
Comment (LuckyPigeon) [reply .size-snapshot.json]: @dai-shi 
Done.

File: .size-snapshot.json [hunk 2]
```diff
@@ -41,24 +41,5 @@
...
-      },
-      "webpack": {
-        "code": 1092
-      }
-    }
-  },
-  "immer.cjs.js": {
-    "bundled": 1228,
-    "minified": 676,
-    "gzipped": 307
```
Comment (dai-shi) [inline .size-snapshot.json:62]: oops, I shouldn't have merged master... sorry. can you deal with it? we don't need `.size-snapshot.json` diff in this PR.

Comment (LuckyPigeon) [reply .size-snapshot.json:62]: @dai-shi 
Or you want me to rebase the `revert the change` commit with the original one?

Comment (dai-shi) [reply .size-snapshot.json:62]: I mean, we can just delete L44-L62 here. Commits are not so important as I will squash them anyway.

Comment (LuckyPigeon) [reply .size-snapshot.json:62]: @dai-shi 
But I already commit it, and L44-L62 is already gone.

Comment (dai-shi) [reply .size-snapshot.json:62]: @LuckyPigeon My apologies. It's the other way around. We need to put back deleted L44-62...

Comment (LuckyPigeon) [reply .size-snapshot.json:62]: @dai-shi 
Roger that & updated. Please check.

**Comment by dai-shi** [comment]
#124 merged.
